README.md

```diff
@@ -15,7 +15,7 @@ You can install this plugin into your CakePHP application using [Composer](http:
 
 Run the following command
 ```sh
-composer require cakephp/migrations
+composer require cakephp/migrations:dev-cake3
```

Comment:
```suggestion
composer require cakephp/migrations:2.x
```
Suggest released versions only.

Reply:
Thanks, got it!

Reply:
Okay, I'm unsure how to proceed though. Do I *commit suggestion* (and if so, will it give credit to you) or do I *add suggestion to batch*?

Reply:
You can use _Commit suggestions_ as there is only one suggestion. _Add suggestion to batch_ is used when there are multiple suggestions.
I will be marked as a co-author of the commit.